source/_integrations/icloud.markdown

```diff
@@ -29,7 +29,7 @@ It does require that your devices are registered with the [Find My](https://www.
 {% include integrations/config_flow.md %}
 
 {% warning %}
-You may receive an email and a notification from Apple saying that someone has logged into your account.
+You may receive recurring emails and notifications from Apple stating that someone has logged into your account if the integration is improperly configured.
```

Comment:
This doesn't help, as it tells the user they did something wrong without providing a solution.

Reply:
Looking to see if I can figure out a method to update the credentials to App-specific Password for the Integration without removing and readding the Integration.

Reply:
Worst case scenario, could we wait for the current auth period to expire?

Reply:
I've updated this in 564a43c. Let me know if you'd like the wording changed/improved.